Sources/Sentry/SentryEventSwiftHelper.m

```diff
@@ -0,0 +1,17 @@
+#import "SentryEventSwiftHelper.h"
+#import "SentryEvent.h"
+#import "SentrySwift.h"
+
+@implementation SentryEventSwiftHelper
```

Comment:
`m`: Why do we need this helper class? It seems so little functionality. Please add some internal documentation